src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -112,10 +114,38 @@ def get(self, request: Request, organization) -> Response:
             return Response(status=404)
 
         projects = self.get_projects(request, organization)
-        try:
-            query = QueryDefinition(request.GET)
-            data = get_datasource(request).get_series(projects, query)
-        except (InvalidField, InvalidParams) as exc:
-            raise (ParseError(detail=str(exc)))
 
-        return Response(data, status=200)
+        def data_fn(offset: int, limit: int):
+            try:
+                query = QueryDefinition(request.GET)
```

Comment:
As discussed offline, let's remove the `limit` parsed inside `QueryDefinition`.

Reply:
See latest commit